ixofoundation/ixo-webclient
# chore: add override info

## Scope
https://nonaredteam.atlassian.net/browse/IXO-852

## Work Done
Added image and text for the demo

## Steps to Test
Check that the override works by visiting: http://localhost:3000/entities/select?type=Cell&categories=[{%22name%22:%20%22Cell%20Type%22,%20%22tags%22:%20[%22Index%22,%20%22Relayer%22,%20%22Portal%22]},%20{%22name%22:%20%22Stage%22,%20%22tags%22:%20[%22Forming%22]}]&sector=Oracle%20Launchpad

## Screenshots
![image](https://user-images.githubusercontent.com/46312800/88193528-0f99d580-cc3e-11ea-8d3b-a0ee4ec3b901.png)
